requirements.sh

```diff
@@ -248,18 +248,20 @@ elif apt --version 2>/dev/null; then
 	apt install -y cmake g++ make build-essential autoconf automake uuid-dev
 	apt install -y libtool libboost-dev libboost-system-dev libboost-thread-dev libpq-dev libz-dev
 	apt install -y python-dev python3-dev python3-pip python3-numpy
+        pip3 install --upgrade pip
 
 	sqlite3_build_prepare
 	make
-
+        
+	apt install -y sqlite3 # make install after sqlite3_build_prepare should be enough to install sqlite3 as a command 
```

Comment:
So we require sqlite3 command in any case?. Only `libsqlite3-dev` is not required as we compile our custom sqlite dev package from https://github.com/dianomic/sqlite3-pkg

Reply:
We should see make install OR make && make install install sqlite3; if that works and handle the cases for create back up and reset etc. We can rely on that and remove pkg dependency in dev env but still the prod. 

I assume libsqlite dev is required for sqlite3 to compile. No harm to keep